NEWS.md

```diff
@@ -1,9 +1,20 @@
 # protti 0.6.0.9000
 
+## New features
+
+* `qc_cvs()` received a new argument called `max_cv` that specifies the maximum CV that should be included in the plot.
+* `peptide_profile_plot()` received a new argument called `complete_sample`. If set to `TRUE`, each protein gets assigned all sample names that are found in the input data. This ensures that the plot always contains all samples on the x-axis even if there are no measured intensities for a specific sample. The default is `FALSE`, which is the original behaviour of the function.
+* `volcano_plot()` received the `colour` argument that allows the user to provide custom colours for points.
+* Increased the speed of `find_peptide()` and `assign_peptide_type()` by only computing on the smallest possible subset of data before joining back to the original data frame.
+
 ## Bug fixes
 
-* `plot_volcano()` now also works interactively if there are no significant hits.
-* `fetch_chebi()` fixed an issue cased by `na_if()` that changed its behaviour after the recent `dplyr` update.
+* `volcano_plot()` now also works interactively if there are no significant hits.
+* `fetch_chebi()`: fixed an issue caused by `na_if()` that changed its behaviour after the recent `dplyr` update.
+* `qc_proteome_coverage()`: fixed the label order of fractions of proteins detected and not detected in the proteome.
+* `calculate_protein_abundance()` now correctly retains columns if `for_plot = TRUE`. Previously the columns to retain were not joined considering the precursor column, which lead to duplications of information where it did not belong.
```

Comment:
I would suggest to link the issue with NEWS.md 
something like this:
* `calculate_protein_abundance()` now ... belong. (#197 ). 

I think it can be really helpful to track back some changes and the reasoning behind them :)

Reply:
Yes totally agree! I was not done with the NEWS.md file yet if I remember correctly. This is also the reason why I did not close some of the issues yet, to remember to put them into the NEWS file. But thanks for commenting!